src/badge/badge.js

```diff
@@ -1,24 +1,36 @@
 import React from 'react';
-import { Text, View, StyleSheet } from 'react-native';
+import { Text, View, StyleSheet, TouchableOpacity } from 'react-native';
 
 let styles = {};
 
 const Badge = props => {
   const { badge } = props;
 
-  if (!badge) throw Error('badge prop is required');
+  const {
+    badgeContainerStyle,
```

Comment:
lets rename them from `badgeContainerStyle` => `containerStyle` and `badgeTextStyle` => `textStyle`